recipes/_source.rb

```diff
@@ -66,11 +67,13 @@
   modules = "--with-modules='#{node['pdns']['authoritative']['backends'].join(' ')}' "
   binary_string = 'pdns_server'
 
-  execute 'pdns: bootstrap' do
-    # This insanity is documented in the README
-    command './bootstrap && ./bootstrap'
-    cwd pdns_source_dir
-    not_if "/usr/local/sbin/#{binary_string} --version 2>&1 | grep #{version}"
+  if major_version < 4
```

Comment:
OK. I think I looked at that possibility briefly and was unclear how `not_if` and `only_if` work together.